### PR TITLE
[ADD] billboard_rental: Billboard Rental Industry

### PR DESCRIPTION
**Pull request by dhrs-odoo**
Task-3522783

### CODE REVIEW

File: billboard_rental/demo/crm_stage.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="crm_stage_5" model="crm.stage">
+        <field name="name">Territory Research</field>
+        <field name="sequence">2</field>
+    </record>
```
Comment (vava-odoo) [inline billboard_rental/demo/crm_stage.xml:6]: To me it could be moved to data

**Review by vava-odoo — CHANGES_REQUESTED** [review]
Thanks Dhrumil. A few comments but overall it looks good. Thanks!

File: billboard_rental/data/ir_model.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="model_worksheet_template_1" model="ir.model">
+        <field name="name">Default Worksheet</field>
+        <field name="model">x_project_task_worksheet_template_1_studio</field>
+    </record>
```
[the remark this answers is not in the document]
Comment (dhrs-odoo) [reply billboard_rental/data/ir_model.xml:6]: yes

File: billboard_rental/data/ir_ui_view.xml [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+                <xpath expr="//div[@name='button_box']" position="after">
+                    <widget name="web_ribbon" title="Busy" bg_color="text-bg-danger" invisible="x_state"/>
+                    <widget name="web_ribbon" title="Available" bg_color="text-bg-success" invisible="not x_state and x_address"/>
+                </xpath>
+                <xpath expr="//field[@name='active']" position="after">
+                    <field name="x_address" widget="res_partner_many2one" context="{'show_address': 1}"/>
+                    <field name="x_city" string="City" invisible="True"/>
+                </xpath>
+                <xpath expr="//field[@name='plan_id']" position="before">
+                    <field name="x_state" string="Available"/>
```
[the remark this answers is not in the document]
Comment (dhrs-odoo) [reply billboard_rental/data/ir_ui_view.xml:106]: It was somehow not getting applied so I have mentioned it over here as well

File: billboard_rental/data/ir_ui_menu.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="billboards_icon" model="ir.ui.menu">
+        <field name="name">Billboards</field>
+        <field name="web_icon">fa fa-desktop,#262c34,#FFFFFF</field>
```
[the remark this answers is not in the document]
Comment (dhrs-odoo) [reply billboard_rental/data/ir_ui_menu.xml:5]: ok

File: billboard_rental/data/sale_order_template_line.xml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        <field name="sale_order_template_id" ref="sale_order_template_1"/>
+    </record>
+    <record id="sale_order_template_line_3" model="sale.order.template.line">
+        <field name="product_id" ref="product_product_4"/>
+        <field name="sale_order_template_id" ref="sale_order_template_2"/>
+    </record>
+    <record id="sale_order_template_line_4" model="sale.order.template.line">
+        <field name="product_id" ref="product_product_5"/>
+        <field name="sale_order_template_id" ref="sale_order_template_2"/>
+    </record>
```
[the remark this answers is not in the document]
Comment (dhrs-odoo) [reply billboard_rental/data/sale_order_template_line.xml:18]: No, this is fine

**Comment by dhrs-odoo** [comment]
Hello @vava-odoo 
I have incorporated all the suggested changes 
Can you please have a look at it once you are free
Thank You :innocent:

File: billboard_rental/demo/sale_order.xml [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        <field name="analytic_account_id" ref="account_analytic_account_5"/>
+        <field name="sale_order_template_id" ref="sale_order_template_1"/>
+    </record>
+    <record id="sale_order_5" model="sale.order">
+        <field name="plan_id" ref="sale_subscription.subscription_plan_month"/>
+        <field name="partner_id" ref="res_partner_14"/>
+        <field name="user_id" ref="base.user_admin"/>
+        <field name="sale_order_template_id" ref="sale_order_template_2"/>
+        <field name="analytic_account_id" ref="account_analytic_account_7"/>
+        <field name="sale_order_template_id" ref="sale_order_template_1"/>
```
Comment (vava-odoo) [inline billboard_rental/demo/sale_order.xml:36]: Duplicated (and runbot was telling you)

File: billboard_rental/data/sale_order_template_line.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (vava-odoo) [reply billboard_rental/data/sale_order_template_line.xml:18]: Short-term:  plan_id: monthly
Long-term: plan_id: yearly

File: billboard_rental/data/uom_uom.xml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="uom_uom_28" model="uom.uom">
+        <field name="name">Months</field>
+        <field name="category_id" ref="uom.uom_categ_wtime"/>
+        <field name="uom_type">bigger</field>
+        <field name="factor">0.03333333333333333</field>
```
[the remark this answers is not in the document]
Comment (vava-odoo) [reply billboard_rental/data/uom_uom.xml:7]: This is not the way. You need to use the reccurring price
image

**Review by vava-odoo — CHANGES_REQUESTED** [review]
Hi @dhrs-odoo 
Thanks for the update. Still some comments before merging.
Cheers

File: billboard_rental/data/sale_order_template_line.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (vava-odoo) [reply billboard_rental/data/sale_order_template_line.xml:18]: Confirmed: plan_id (same as for sale.subscription.pricing) to be added in the sale.order.template

[the remark this answers is not in the document]
Comment (dhrs-odoo) [reply billboard_rental/data/sale_order_template_line.xml:18]: Done added the plan_id in the sale.order.template

**Pull request merged**